code/src/index.css

```diff
@@ -11,3 +11,171 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+body {
+  background-image: url(https://img.freepik.com/free-vector/linear-flat-abstract-lines-pattern_23-2148944144.jpg?w=740&t=st=1679234138~exp=1679234738~hmac=77ea19501fff6a0b8d30afafdb0fb128a8565547be77430806cc0d20fe6363ec);
+  background-repeat:no-repeat;
+  background-size:cover;
+  background-attachment: fixed;
+}
+
+
+.survey-container {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  align-items: center;
+  justify-content: center;
+  background-color:rgba(239, 154, 181, 0.458);
+  height: 600px;
+  margin-top: 50px;
+  margin-left: 25px;
+  margin-right: 25px;
+  border-radius: 10px;
+
```

Comment:
So, I won't comment a ton on your CSS - you've already said you're updating it.  But, I wanted to share "vw" and "vh" as measurements for view width and view height.  If you've already experimented with these, ignore me.  They were new to me as I was styling this project and were SUPER helpful in forcing my survey to take up the amount of space that I wanted it to.